src/resolvers/network.resolver.ts

```diff
@@ -18,20 +19,27 @@ export class NetworkResolver implements Resolver {
      * @param {string} altText - Alternative text.
      * @returns {number}
      */
-    resolve(options: ProfileOptions, secondSource?: Profile, altText?: string): any {
-        const choices = ['MAIN_NET', 'TEST_NET', 'MIJIN', 'MIJIN_TEST'];
-        const resolution = OptionsChoiceResolver(options,
+    async resolve(options: ProfileOptions, secondSource?: Profile, altText?: string): Promise<any> {
+        const choices = [
+            {title: 'MAIN_NET', value: 0},
+            {title: 'TEST_NET', value: 1},
+            {title: 'MIJIN', value: 2},
+            {title: 'MIJIN_TEST', value: 3},
+        ];
+        const index = +(await OptionsChoiceResolver(options,
             'network',
             altText ? altText : 'Select the network type: ',
             choices,
-        );
-        let networkFriendlyName;
-        if (isNumeric(resolution)) {
-            networkFriendlyName = choices[+resolution] as any;
-        } else {
-            networkFriendlyName = resolution;
+        ));
+        const networkFriendlyName = choices.find((item) => {
```

Comment:
If we pass an option in the command line, it will never work because the index will be a string. Correct me if I'm wrong (applies to other resolvers using ``OptionsChoiceResolver``.